internal/services/containers/kubernetes_cluster_addons_resource_test.go

```diff
@@ -24,7 +24,7 @@ func TestAccKubernetesCluster_addonProfileAciConnectorLinux(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("public_network_access_enabled"),
```

Comment:
rather than doing this, we can set a default value for this field in the `import` function - else we'll flag a diff for users doing a regular import - so can we update this

Reply:
Thanks! I've updated this PR, please take another look.